### PR TITLE
Update ini files and particles for benchmarks

### PR DESCRIPTION
**Pull request by awirb**
* new topo names - update benchmark files for com1DFAPy
* update docu for com1DFA

### CODE REVIEW

**Review by matthiasto — APPROVED** [review]

File: benchmarks/avaHockeyChannelentTest/avaHockeyChannel_com1DFACfgPy.ini [hunk 1]
```diff
@@ -110,7 +110,7 @@ tSteps = 0|-1
 #++++++++++++++++ particle Initialisation
 # if true use file (import particles initial distribution from file)
 initialiseParticlesFromFile = True
-particleFile = ../benchmarks/avaHockeySmoothChannelentTest
+particleFile = ../benchmarks/avaHockeyChannelentTest
```
Comment (fso42) [inline benchmarks/avaHockeyChannelentTest/avaHockeyChannel_com1DFACfgPy.ini:113]: Small change, if it is not too late:
```suggestion
particleFile = ../benchmarks/avaHockeyChannelEntTest
```

File: benchmarks/avaHockeySmallnullTest/avaHockeySmall_com1DFACfgPy.ini [hunk 2]
```diff
@@ -110,7 +110,7 @@ tSteps = 0|-1
 #++++++++++++++++ particle Initialisation
 # if true use file (import particles initial distribution from file)
 initialiseParticlesFromFile = True
-particleFile = ../benchmarks/avaHockeySmoothSmallnullTest
+particleFile = ../benchmarks/avaHockeySmallnullTest
```
Comment (fso42) [inline benchmarks/avaHockeySmallnullTest/avaHockeySmall_com1DFACfgPy.ini:113]:
```suggestion
particleFile = ../benchmarks/avaHockeySmallNullTest
```

File: benchmarks/avaParabolaresTest/avaParabola_com1DFACfgPy.ini [hunk 3]
```diff
@@ -110,7 +110,7 @@ tSteps = 0|-1
 #++++++++++++++++ particle Initialisation
 # if true use file (import particles initial distribution from file)
 initialiseParticlesFromFile = True
-particleFile = ../benchmarks/avaHockey/particles/release1HS_entres_dfa_0.15500
+particleFile = ../benchmarks/avaParabolaresTest
```
Comment (fso42) [inline benchmarks/avaParabolaresTest/avaParabola_com1DFACfgPy.ini:113]:
```suggestion
particleFile = ../benchmarks/avaParabolaResTest
```

**Review by fso42 — CHANGES_REQUESTED** [review]

File: benchmarks/avaHockeyChannelentTest/avaHockeyChannel_com1DFACfgPy.ini (hunk 1 above)
Comment (awirb) [reply benchmarks/avaHockeyChannelentTest/avaHockeyChannel_com1DFACfgPy.ini:113]: Done

File: benchmarks/avaHockeySmallnullTest/avaHockeySmall_com1DFACfgPy.ini (hunk 2 above)
Comment (awirb) [reply benchmarks/avaHockeySmallnullTest/avaHockeySmall_com1DFACfgPy.ini]: ok

**Review by fso42 — APPROVED** [review]

**Pull request merged**